### PR TITLE
Add LeaderfQuickFix and LeaderfLocList

### PR DESCRIPTION
**Pull request by tamago324**
The following two commands have been added

* LeaderfQuickFix
* LeaderfLocList

Refer #558 

Thank you!

### CODE REVIEW

File: doc/leaderf.txt [hunk 1]
```diff
@@ -986,6 +986,12 @@ USAGE                                           *leaderf-usage*
 :LeaderfWindow                                  *LeaderfWindow*
     Launch LeaderF to search windows.
 
+:LeaderfQuickFix                                *LeaderfQuickFix*
```
Comment (Yggdroot) [inline doc/leaderf.txt:989]: Seems these two commands are not defined yet.

Comment (tamago324) [reply doc/leaderf.txt:989]: I'm sorry.
Added.

**Comment by Yggdroot** [comment]
Thanks.

**Comment by tamago324** [comment]
Thank you!

**Pull request merged**